test/subdataframe.jl

```diff
@@ -96,8 +96,8 @@ module TestSubDataFrame
     end
 
     @testset "getproperty, setproperty! and propertynames" begin
-        x = 1:10
-        y = 1.0:10.0
+        x = collect(1:10)
```

Comment:
Is this actually needed? In my tests this doesn't make any difference.

Reply:
It does - as below `x` is mutated and we do `@test x == [1; 1:5; 7:10]` several lines below. The same for `y`.